python/tests/test_lifting.py

```diff
@@ -113,8 +114,8 @@ def l2b(li):
             reduced_L = K.T @ (L_np)  # - constants)
             # Solve linear system
             d = scipy.sparse.linalg.spsolve(KTAK, reduced_L)
-            # Back substitution to full solution vecto
+            # Back substitution to full solution vector
             uh_numpy = K @ (d)  # + constants)
-            assert np.allclose(uh_numpy, u_mpc)
+            nt.assert_allclose(uh_numpy, u_mpc, rtol=1e-5, atol=1e-8)
```

Comment:
This was the only one that used default tolerances, which differ. allclose has 1e-5, 1e-8 while assert_allclose has 1e-7, 0.